quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMappers.java

```diff
@@ -367,10 +336,13 @@ public Map<String, PropertyMapper<?>> getDisabledRuntimeMappers() {
         }
 
         private static void sanitizeMappers(Map<OptionCategory, List<PropertyMapper<?>>> mappers,
-                                            Map<String, PropertyMapper<?>> disabledMappers) {
+                                            Map<String, PropertyMapper<?>> disabledMappers, AbstractCommand command) {
             mappers.forEach((category, propertyMappers) ->
                     propertyMappers.removeIf(pm -> {
-                        final boolean shouldRemove = !pm.isEnabled();
+                        // do a localized check based upon the current command
+                        // to exclude one or both of the import / export options as they overlap with one another
+                        final boolean shouldRemove = !pm.isEnabled() || (pm.getCategory() == OptionCategory.IMPORT && !(command instanceof Import))
```

Comment:
Would it be possible to have some general approach to check the category for the mapper and used command? Leveraging `AbstractCommand.getOptionCategories()`?